tests/jsx/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,5 +1,84 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`conditional-expression.js 1`] = `
+<div>
+  {isCool ? "cool" : "very nice"}
+</div>;
+
+// should get formatted JSX style
+const a = something ? "string" : <div />;
+
+// should not get formatted JSX style
+const a = something ? "string" : "other string";
+
+<div>
+  {reallyLongChainOfConditionalExpressions ? (
+    "why"
+  ) : yes ? (
+    "it"
+  ) : is ? (
+    "!"
+  ) : (
+    ":)"
+  )}
+</div>;
+
+<div>
+  {reallyLongChainOfConditionalExpressions ? (
+    <span>why</span>
+  ) : yes ? (
+    <span>it</span>
+  ) : is ? (
+    <span>!</span>
+  ) : (
+    <span>:)</span>
+  )}
+</div>;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<div>
+  {isCool ? (
```

Comment:
The way to solve this is to "linearize" the nested conditionals. You can look at printMemberChain for an example.

Once it's in a flat array, it's easier to put only a single group around, to check if there's a JSX element inside...

We'll need to get this in before the release (doesn't have to be in this PR), otherwise we're going to churn people where we change their code in one way, and then back to another.